### DISCUSSION TITLE
デバッグ募集

### DISCUSSION THREAD

**Discussion by i7-FF7FC** [beta1139/ff7fc-develop#88] [General]
前回の続きからカーム回想の前半のイベントまで作成しました。
あまり長くはありませんがよろしくお願いします:smiley:
ponyexsさんから頂いたseは、「spring.ogg」と「magic_light_reactor_pod.ogg」
という名前で実装しています。

いやぁひたすら画像ばっか打ってて疲れた:persevere:

**Comment by ponyexs** [comment]
回想イベント制作お疲れ様です。

そして効果音のご採用ありがとうございます。
実際にゲーム内で鳴っているところを聞くと感無量ですね

ちょっとせのびパンツ、ついに「具体的」に実装されたのですねw
説明文のセンスが特にすばらしいです。興味津々なのに感心の無いフリ、をクラウドらしさ全開で表すあのセリフ！

そんなお年頃のクラウド君を堪能しつつ日課のピアノ演奏をしてニブル山へ出かけました。

吊り橋が歩くたびに揺れるのがリアル！
バルブを斬りつけるたびに揺れるのもの凝ってますね！さすがのクオリティです！
冷却装置内のモンスターのイラストも凄いですね！
しかも装置からモンスターが飛び出してくるムービーも再現まで！さすが！
マップの再現度の高さももちろん！
この凄さと感動を表す語彙が足りない自分の感性が残念です。

---
デバッグということで、ニブル山をうろついで気が付いたことをIssueに記載します。


…ところでこのような場合ですとデバッグへの応答ということで
もしかしたらこちらに続けて記載したほうが確認の手間が省けていいんでしょうか？

**Reply by i7-FF7FC** [reply]
ありがとうございます！:blush:

> 実際にゲーム内で鳴っているところを聞くと感無量


凄く分かります。
私も初めて自分が打ったスクリプトでクラウド達が動いた時
めちゃくちゃ感動して気が付いたら現在に至る感じですのでw


> ちょっとせのびパ

早速バレてしまうなんて！:persevere:
やはりあの短い瞬間に落とすのはあのパワーワードしかないかなとw

> 吊り橋が

橋もそうですが、今回はスクリプトというより、
ほぼほぼ不得手な画像作成との連戦で大変でした。。
特に魔晄炉のシーンは、ストーリー上かなり重要な場面なので、
出来るだけ丁寧にやってみたかったので、なんとか形にできてホッとしています。
モンスターは当初ストⅡの豪鬼みたいなのをイメージしていたんですが、
気が付いたらFF10のウェンディゴみたいになっちゃいました:joy_cat:



> 装置からモンスターが飛び出してくるムービーも再現まで！

あれ当初蓋が開いた感じが弱くて、
わざわざポッドが開いた時の斜め上の画像用意して、別途間に挟んだんですよ。
そしたらおー開いとる開いとる！みたいになって、
一人で感動しつつやってたので共有できてうれしい限りです！



> こちらに続けて記載したほうが確認の手間が省けて

今回のイベントに関するデバッグ報告は、基本こちらでしてもらって構いません。
寧ろスレッドネタもないと思うので、こちらでお願いしますー:dash:

**Reply by ponyexs** [reply]
橋の揺れ、スプリクトなどでの制御かと思ってましたが
画像を描いているんですね！

ほんとお疲れ様です。
そしてその心配りから生まれている高品質な作品をありがとう！


> 今回のイベントに関するデバッグ報告は、基本こちらでしてもらって構いません

おっとっと先にIssueに書いちゃったんで、
では次の同じようなモノが見つかったら今度こそここにっ😤

**Reply by i7-FF7FC** [reply]
ああ誤解を招いたようですみません:sweat_drops:
揺れ自体はスクリプトなんですが、橋や落ちる画像、崩れかける画像とか
そういうのも諸々用意する必要があったので大変でした、という意味でした。
しかしながらponyexsさんが色々気付いて下さるので、作り甲斐がありますよw
こちらこそ改めて感謝です。SEも素晴らしいものをありがとうございます！:thumbsup:

**Comment by i7-FF7FC** [comment]
マップ投下
ジェノバルーム
カーム関連はたぶんこれで最後かな:question:
![324](https://user-images.githubusercontent.com/11593576/111164642-99d5e600-85e1-11eb-9171-6fa832ea7ebb.png)

**Comment by ponyexs** [comment]
うお凄い！リアル！ジェノバだけでなくタイルと柱が特に！

2021年3月15日(月) 23:00 i7-FF7FC ***@***.***>:

**Reply by i7-FF7FC** [reply]
ありがとうございます！
ジェノバ像を打てるか半信半疑だったんですけど、何とか形に出来てホットしてます。。:relieved:
個人的には像の背後にあるメディカルマシーンみたいなのが、割とイメージ通りになって気に入っていますー。
あと元々床にJENOVAのプレートも置いてたんですけど、整理したら思いのほかチップ使ってたんで消えましたw

残すはここ関連のスクリプトのみ・・・:dash:

**Comment by i7-FF7FC** [comment]
カーム回想粗方終わったんで、
確認しつつ近いうちアップできると思います。
演出とか色々考えてたら、結構時間かかってしまった。

**Reply by ponyexs** [reply]
制作ほんとにお疲れ様です🙇
楽しみにしてますー😍😍

**Reply by i7-FF7FC** [reply]
ありがとうございますー。
やっとカームから脱出できそうですw

あ　以前頂いた魔晄炉内のBGM有難く使わせてもらいました！
今回色々手一杯だったので本当に助かりました:joy::pray:ありがとうございます！

**Comment by i7-FF7FC** [comment]
カーム回想後半イベント作成しました:smiley:
デバッグよろしくお願いしますー。

Those Chosen~のBGMは以前言っていたように、一応2パターン用意してみました。
屋敷の地下書斎の入り口付近に切り替えスイッチ用意しています。
改めて判定のほど...:ok_woman::no_good:

**Reply by ponyexs** [reply]
こんばんは、制作お疲れ様です。
早速遊ばせてもらいました🤩


セフィロスの引きこもりの日々がすごいですね。
流れるテロップの横でフェードアウトして移動するセフィロスと通路に本が積み重なっていく演出がリアル🎉


炎の中の顔、にやりと笑うセフィロス、そして縦に真っ二つかっこいい！
そして魔晄炉で対峙するシーンの演出も！


そして語り終えた後の締めの一言には笑いました
「というか作ってない」

遊び心がうれしいです


そうそうセフィロスがジェノバ像に話しかけるシーン、
3-05-Those_Chosen_by_the_Planetのフルバージョンも用意されたんですね。

…と思ったんですが、BGMフォルダに見当たりませんね。
探し方が悪いのかな…🤔

…それとも、もしかしてゲーム側の処理で調整して
EX-12-Pre_Those_Chosenを鐘4回で止めるのを2回行った後
本編の3-05-Those_Chosen_by_the_Planetを再生、
というように表現してたりします？？



切り替えBGMですが、
鐘の音はパターン1のほうが存在感がありますけれど
この不気味なシーンにはパターン2のほうが馴染むような感じを受けました。


バグ関連ですが、
今回の範囲では発見はありませんでした。
そして神羅屋敷を取り囲む人たちも含めた再現度もさすがっ！😤


--------------------------------------------------------------------
…ただ実のところ、今回新シーンじゃなく
従来の範囲で遭遇した現象がありましたので
Issueのほうへ載せます。

**Reply by i7-FF7FC** [reply]
今回も色々気付いていただいて滅茶苦茶嬉しい限りです。。:innocent:

> フェードアウトして移動するセフィロス

あのマップ残しでキャラのみフェードアウトorマップのみフェードアウトは、
前回の魔晄炉イベで開発した新技()です。
今回も偶然使える場面があったのでちょうど良かったです。


> 鐘4回で止めるのを2回行った後


ですです！
前奏の時間から凡そのフレーム数を算出して
イベントと同期させています:smirk:


> この不気味なシーンにはパターン2のほうが馴染むような感じを受けました。


あーやっぱですよね、、私も悩んでて。
当初ほぼ1で決まりだろうということで、
申し訳程度にもう1パターン用意してみたつもりだったんですがw
意外とヒュードロドロ感:skull:が出て、
これは逆にアリかも知れないと思い始めてw

**Reply by ponyexs** [reply]
>前奏の時間から凡そのフレーム数を算出して
>イベントと同期させています😏

凄いですね！ほんとにスクリプトで曲の再現をされているんですね
発想がさすが！

**Comment by i7-FF7FC** [comment]
お久しぶりです。
一応YouTube動画一つ上げてみました。
https://www.youtube.com/watch?v=jkRjUv_BpJM
ただプレイするだけなのもアレだったので、
昔作ったOPをちょっと弄って追加しときました:smile:
(他既出なので見所そこだけです)

**Reply by beta1139** [reply]
素晴らしい！！

**Reply by ponyexs** [reply]
OP凄い！

**Reply by i7-FF7FC** [reply]
あざます！:blush:
そのうちゲーム内からも飛べるようコミットしときます。

さてチョコボ牧場どうしようか考え中。。
森で捕まえるようにするなら、牧場イベどうしたらいいんだろ:thinking:
